src/Hazelcast.Net.Examples/DistributedObjects/MapCustomSerializerExample.cs

```diff
@@ -25,6 +25,8 @@ public class MapCustomSerializerExample
     {
         public static async Task Main(string[] args)
         {
+            Console.WriteLine("NOTE: BinaryFormatter is obsolete and should not be used. This example is for demonstration purposes only.");
```

Comment:
Why use binaryformatter in the code sample if it is bad practice?

Reply:
It works on other targets -> .net6, .net7, .net48, .net462